web/views/bundles_view.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Cog.V1.BundlesView do
+  use Cog.Web, :view
+
+  alias Ecto.Association
+
+  def render("bundle.json", %{bundle: bundle}) do
+    %{id: bundle.id,
+      name: bundle.name,
+      namespace: bundle.namespace.name,
+      commands: render_commands(bundle.commands),
+      relay_groups: render_relay_groups(bundle.relay_groups),
+      enabled: bundle.enabled,
+      inserted_at: bundle.inserted_at,
+      updated_at: bundle.updated_at}
+  end
+  def render("index.json", %{bundles: bundles}) do
+    %{bundles: render_many(bundles, __MODULE__, "bundle.json", as: :bundle)}
+  end
+  def render("show.json", %{bundle: bundle}) do
+    %{bundle: render_one(bundle, __MODULE__, "bundle.json", as: :bundle)}
+  end
+
+  def render_relay_groups(%Association.NotLoaded{}) do
```

Comment:
Both index and show delegate to `bundle.json`. The only difference is whether they return a single bundle or a list of bundles. I think that's mostly a guard to keep from blowing up if the association isn't loaded for some reason, but we should probably crash instead of returning a possibly inaccurate empty list of `relay_groups`.

I'll make that change.